art/attacks/inference/membership_inference/label_only_boundary_distance.py

```diff
@@ -196,6 +202,70 @@ def calibrate_distance_threshold(
 
         self.distance_threshold_tau = distance_threshold_tau
 
+    def calibrate_distance_threshold_unsupervised(
+        self, top_t: int = 50, num_samples: int = 100, max_queries: int = 1, **kwargs
+    ):
+        """
+        Calibrate distance threshold on randomly generated samples, choosing the top-t percentile of the noise needed
+        to change the classifier's initial prediction.
```

Comment:
Can you please add to the method description a statement that says that it requires the model's clip_values to be set? Thanks.

Reply:
Done!